usaspending_api/download/tests/integration/test_download_status.py

```diff
@@ -204,7 +204,7 @@ def test_download_contract_status(client, download_test_data, refresh_matviews):
 
     assert resp.status_code == status.HTTP_200_OK
     assert resp.json()["total_rows"] == 1
-    assert resp.json()["total_columns"] == len(query_paths["transaction"]["d1"])
```

Comment:
Why was this changed? Contracts are FPDS aka D1, IDVs are also from FPDS but they are not awards

Reply:
That change was to have it match the slightly longer map during a period of commits where idv_transaction_history had the column while transaction still didn't. With a later change I realized it needed to be on transaction to, bringing these dicts to the same length and making this test a false negative.